mptt/managers.py

```diff
@@ -794,46 +553,34 @@ def _inter_tree_move_and_close_gap(
         connection = self._get_connection(instance=node)
         qn = connection.ops.quote_name
 
-        opts = self.model._meta
         inter_tree_move_query = """
         UPDATE %(table)s
```

Comment:
I'm no expert on the internals of django-mptt, but why is raw sql needed here? Could you do:

```python
Model.objects.update(
    level=Case(When(lft__gte=x, then=F('level') - ....))),
    tree_id=Case(When(lft__gte=y...
)
```

Using the Case/When classes built into Django?